test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
+    ?a
+  end
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+l[x]
+l.call(x)
+
+  def b(x)
+    
+  end 
+
+
+  ['x','b',"c"]
```

Comment:
__[development][rubocop]__
* Literal `[x, a, b, c, d, e]` used in void context.
* Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
* Inconsistent indentation detected.
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->